src/pages/GardenAdmin/GardenSyncAllButton.tsx

```diff
@@ -0,0 +1,76 @@
+import LoadingButton from '@mui/lab/LoadingButton'
```

Comment:
Was this an abandoned plan? Because this doesn't appear to be used anywhere (or else I'm missing something).

Reply:
I abandon it cause I realized the sync button that was already made would work without any changes and forgot that I added that file.